src/main/kotlin/no/nav/familie/ef/sak/regler/evalutation/OppdaterVilkår.kt

```diff
@@ -0,0 +1,78 @@
+package no.nav.familie.ef.sak.regler.evalutation
+
+import no.nav.familie.ef.sak.api.Feil
+import no.nav.familie.ef.sak.api.dto.DelvilkårsvurderingDto
+import no.nav.familie.ef.sak.api.dto.svarTilDomene
+import no.nav.familie.ef.sak.regler.Vilkårsregel
+import no.nav.familie.ef.sak.regler.Vilkårsregler.Companion.VILKÅRSREGLER
+import no.nav.familie.ef.sak.regler.evalutation.RegelEvaluering.utledResultat
+import no.nav.familie.ef.sak.regler.evalutation.RegelValidering.validerVurdering
+import no.nav.familie.ef.sak.repository.domain.DelvilkårsvurderingWrapper
+import no.nav.familie.ef.sak.repository.domain.VilkårType
+import no.nav.familie.ef.sak.repository.domain.Vilkårsresultat
+import no.nav.familie.ef.sak.repository.domain.Vilkårsvurdering
+
+object OppdaterVilkår {
+
+    /**
+     * Oppdaterer [Vilkårsvurdering] med nye svar og resultat
+     * Validerer att svaren er gyldige
+     */
+    fun lagNyOppdatertVilkårsvurdering(vilkårsvurdering: Vilkårsvurdering,
+                                       oppdatering: List<DelvilkårsvurderingDto>,
+                                       vilkårsregler: Map<VilkårType, Vilkårsregel> = VILKÅRSREGLER.vilkårsregler): Vilkårsvurdering {
+        val vilkårsregel = vilkårsregler[vilkårsvurdering.type] ?: error("Finner ikke vilkårsregler for ${vilkårsvurdering.type}")
+
+        validerVurdering(vilkårsregel, oppdatering)
+
+        val vilkårsresultat = utledResultat(vilkårsregel, oppdatering)
+        validerAttResultatErOppfyltEllerIkkeOppfylt(vilkårsresultat)
+        val oppdaterteDelvilkår = oppdaterDelvilkår(vilkårsvurdering, vilkårsresultat, oppdatering)
+        return vilkårsvurdering.copy(resultat = vilkårsresultat.vilkår,
+                                     delvilkårsvurdering = oppdaterteDelvilkår)
+    }
+
+    /**
+     * Då vi ikke helt har støtte for [Vilkårsresultat.SKAL_IKKE_VURDERES] ennå så skal svaret være
+     * [Vilkårsresultat.IKKE_OPPFYLT] eller [Vilkårsresultat.OPPFYLT]
+     * Når man legger til funksjonalitet for SKAL_IKKE_VURDERES, hva skal resultatet være?
```

Comment:
Tror vi må støtte at svaret er `SKAL_IKKE_VURDERES`, men vet ikke helt hvordan det blir i "logikken" ellers. 
Kanskje en default hvisSkalIkkeVurderes => SLUTT_NODE ? 🤔

Reply:
Alternativt må det settes på "topp" på samme måte som "nullstillVilkår" - mulig det forenkler litt.

Reply:
Ja, det er bare att jeg ikke helt vet hva loggikken i øvrig skal være så unngikk denne nå :) 
Men det skal vel være mulig å legge til støtte for den